include/generators/lp_generator.h

```diff
@@ -0,0 +1,119 @@
+//
```

Comment:
Please use the library-copyright-licence comments as in the begining of the other files of the library.
You can add a message like 
`//Contributed and/or modified by Panagiotis Repouskos, as part of Google Summer of Code 2019 program.`